environment.yml

```diff
@@ -15,5 +14,5 @@ dependencies:
   - sphinx_rtd_theme
   - sphinxcontrib-apidoc
   - sqlalchemy
-  - antlr4-python3-runtime =4.9.2
+  - antlr-python-runtime==4.9.2
```

Comment:
Was changing it here to `antlr-python-runtime` and keeping it `antlr4-python3-runtime` in `setup.cfg` by design or by accident?

Reply:
This is intentional. The file lists conda dependencies, and `antlr-python-runtime` is the name of the more popular conda package. While conda-forge does have a package named `antlr4-python3-runtime` too, I have been told that it is deprecated.

Meanwhile the only name available on PyPI is `antlr4-python3-runtime`.

Reply:
Good to know, thanks!